spec/services/proofing/lexis_nexis/instant_verify/verification_request_spec.rb

```diff
@@ -98,4 +98,15 @@
       end
     end
   end
+
+  describe '#formatted_address' do
+    context 'when ZIP code is nil' do
+      it 'returns nil without raising an exception' do
+        applicant[:zipcode] = nil
+
+        expect { subject.send(:formatted_address) }.not_to raise_error(NoMethodError)
```

Comment:
I would remove this? since the same exact thing passes below, that means it's not raising a NoMethodError
```suggestion
```